### DISCUSSION TITLE
无法挂载“相册”

### DISCUSSION THREAD

**Discussion by icewin** [messense/aliyundrive-webdav#332] [Q&A] [answered]
挂载的根目录是“文件”目录，但阿里云盘的相册不在此目录下，所以无法挂在。
但平时手机最常用的阿里云盘功能就是相册自动备份，又怕不安全，想在nas上从aliyunpan再备份一份下来。（aliyunpan 的备份都是原片，群晖moments的备份都是压缩处理，且效率贼慢……）
请问，目前有没有规避方案能够实现这个需求？

**Comment by messense** [comment, marked as answer]
没有：https://github.com/messense/aliyundrive-webdav/issues/95
